.style.yapf

```diff
@@ -15,7 +15,7 @@ SPACES_BEFORE_COMMENT=2
 SPACE_BETWEEN_ENDING_COMMA_AND_CLOSING_BRACKET=False
 SPLIT_ARGUMENTS_WHEN_COMMA_TERMINATED=False
 SPLIT_BEFORE_BITWISE_OPERATOR=True
-SPLIT_BEFORE_FIRST_ARGUMENT=True
+SPLIT_BEFORE_FIRST_ARGUMENT=False
```

Comment:
That's the problem with style, everyone thinks slightly differently, so I'd like to just adopt the google standard as closely as possible. I made this change to make it align better with google style so my editor set to enforce google style doesn't fight with yapf. I've asked the author if he can supply a yapf style file that mimicks internal style, we should adopt that if one exists. If not, this change at least fixed the most obvious difference.